dts/bindings/dma/st,stm32-dma.yaml

```diff
@@ -30,3 +38,8 @@ dma-cells:
   - slot
   - channel-config
   - features
+
+# channel: DMA channel
+# slot: DMA stream of the DMA channel for DMA V1 or MUX peripheral request for DMA V2
```

Comment:
"MUX peripheral request for DMA V2"
Some V2 don't use DMAMUX, please rework the comment.

Reply:
slot: DMA stream of the DMA channel for DMA V1 or for DMA V2 with MUX peripheral request else NA